2-projects/1-todo-list/script.js

```diff
@@ -17,9 +57,25 @@ function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+   let inputs = document.getElementById("todoInput");
+let addI = inputs.value;
+if (!addI) {
+  return;
+} todos.push({
+  task: addI,
+  completed: false,
+});
+populateTodoList(todos);
+inputs.value != "";
```

Comment:
!= is used to compare the left side with the right side
What were you trying to do on this line?